crypto/provider_core.c

```diff
@@ -103,6 +104,7 @@ static void *provider_store_new(OPENSSL_CTX *ctx)
         provider_store_free(store);
         return NULL;
     }
+    store->libctx = ctx;
```

Comment:
You aren't using this anywhere as far as I can see.  I suppose that was earlier experimentation?

Reply:
Hmmm. I don't recall why I added it - but you're right. It doesn't seem to be needed any more so I've removed it.